internal/utils/navitia_datetime.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"regexp"
+	"time"
+
+	"github.com/araddon/dateparse"
+)
+
+func PraseNavitiaDateTime(datetime string, location string) (time.Time, error) {
+	// Parse datetime accepted by navitia: https://doc.navitia.io/#standard-objects
+
+	// datetime with format: YYYYMMDDThhmmss
+	loc, _ := time.LoadLocation(location)
+	t, err := time.ParseInLocation("20060102T150405", datetime, loc)
+	if err == nil {
+		return t.UTC(), nil
+	}
+	// datetime with format: YYYYMMDDThhmmssZ
+	dtf := regexp.MustCompile(`([0-9]{4})([0-9]{2})([0-9]{2})T([0-9]{2})([0-9]{2})([0-9]{2})(Z)`)
```

Comment:
(same for other code occurrences)
minor:
```suggestion
	dtf := regexp.MustCompile(`(\d{4})(\d{2})(\d{2})T(\d{2})(`d{2})(\d{2})Z`)
```
:point_up: Also, you can avoid parenthesis around the Z in this case (not reusing it).